acceptance/acceptance_suite_test.go

```diff
@@ -280,8 +295,11 @@ func installEpinio() (string, error) {
 	return Epinio("install --skip-default-org", "")
 }
 
-func uninstallEpinio() (string, error) {
-	return Epinio("uninstall", "")
+// Remove all tmp directories from /tmp/epinio-* . Test should try to cleanup
+// after themselves but that sometimes doesn't happen, either because we forgot
+// the cleanup code or because the test failed before that happened.
```

Comment:
Since this PR also fixes the leaking tests, we're not sure if we  need that extra clean up?
Is the "that sometimes doesn't happen" part still true?

I would add a warning:

> This code will create problems if more than one acceptance_suite_test.go is run in parallel (e.g. two PRs on one worker). However we keep it as an extra measure.

Reply:
added